style.css

```diff
@@ -9,6 +9,7 @@
   --surface: #c0c0c0;
   --button-highlight: #ffffff;
   --button-face: #dfdfdf;
+  --button-text: #000;
```

Comment:
I think we want #222222 https://github.com/jdan/98.css/pull/141/files#diff-b78be019f1dc6d57753ea900c3805b114cd53ab7c0db836cc081836df1b99b7aL92

Reply:
maked :))